modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/NestedChainModelParser.java

```diff
@@ -10,13 +10,13 @@
 import org.mule.runtime.api.meta.model.nested.NestedChainModel;
 
 /**
- * Parses the syntactic definition of a {@link NestedChainModel} so that the semantics reflected in it can be extracted in a
+ * Parses the syntactic definition of a {@link NestedChainModel} so that the semantics reflected in it can be extracted in an
```

Comment:
It seems it was correct before, the 'u' sound there is not considered a vowel apparently.